example-apps/Token-EasySending/src/components/SendTokenModal.vue

```diff
@@ -316,7 +315,12 @@ const onSubmit = async (_e: Event) => {
       sendStatus.value = "processing";
       message.value = "Check your wallet to confirm the operation...";
 
-      const target = ethers.utils.getAddress(transaction.value.to);
+      const target = validateEmail(transaction.value.to) && ethers.utils.getAddress(
+        await getAccountAddress(
+          createNameStruct(transaction.value.to, "email")
```

Comment:
May I ask what is the difference between this function and getName() in account.ts